L1Trigger/L1TCalorimeter/src/firmware/Stage2Layer2JetAlgorithmFirmwareImp1.cc

```diff
@@ -157,20 +157,23 @@ void l1t::Stage2Layer2JetAlgorithmFirmwareImp1::create(const std::vector<l1t::Ca
 	    int caloEta = CaloTools::caloEta(ieta);
 	    l1t::Jet jet( p4, -999, caloEta, iphi, 0);
 
-	    if (PUSubMethod == "Donut") {
-	      puEt = donutPUEstimate(ieta, iphi, 5, towers);	    
-	      iEt -= puEt;
+	    // remove checking if(!params_->jetBypassPUS()). jetBypassPUS_ is currently always set and configured to false
+	    if(true){
+	      if (PUSubMethod == "Donut") {
+		puEt = donutPUEstimate(ieta, iphi, 5, towers);	    
+		iEt -= puEt;
+	      }
+	      
+	      if (PUSubMethod == "ChunkyDonut"){
+		puEt = chunkyDonutPUEstimate(jet, 5, towers);
+		iEt -= puEt;
+	      }
 	    }
 	    
-	    if (PUSubMethod == "ChunkyDonut"){
-	      puEt = chunkyDonutPUEstimate(jet, 5, towers);
-	      iEt -= puEt;
-	    }
-
 	    if (iEt<=0) continue;
 
 	    // if tower Et is saturated, saturate jet Et
-	    if (seedEt >= 511) iEt = 65535;
+	    if (seedEt >= 509) iEt = 65535;
```

Comment:
In this 80x (back-port version of #16324) the jet emulator uses updated tower-saturation points (509, 510, and 511), as already commented in 81x version (https://github.com/cms-sw/cmssw/pull/16324#discussion_r84743726)